src/test/java/teammates/test/cases/browsertests/StudentFeedbackResultsPageUiTest.java

```diff
@@ -79,7 +79,7 @@ public void testAll() throws Exception {
 
         ______TS("MSQ session results");
 
-        resultsPage = loginToStudentFeedbackResultsPage("Alice", "MSQ Session");
+        resultsPage = loginToStudentFeedbackResultsPage("Benny", "MSQ Session");
```

Comment:
How does this change help?

Reply:
Instead of having all checks only for Alice, I made some even change to check from Benny. This will be helpful as it will cause GodMode changes when certain functionalities of questions are changed